docs/content/docs/tutorials/sqs-example.md

```diff
@@ -44,15 +44,18 @@ This guide assumes that you have:
 
 ### Install the ACK service controller for SQS
 
+> **_NOTE:_** This guide assumes you're using `us-east-1` as the region where the ACK controller will be deployed, as well as the Amazon SQS resource. If you want to create the object in another resource, simply change the region name to your region of choice.
+
 Log into the Helm registry that stores the ACK charts:
 ```bash
-aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+CONTROLLER_REGION=us-east-1
+aws ecr-public get-login-password --region $CONTROLLER_REGION | helm registry login --username AWS --password-stdin public.ecr.aws
```

Comment:
The ECR public endpoint is only available in us-east-1

Reply:
Thank you. Just fixed that